sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMiningAsync.cs

```diff
@@ -33,33 +34,33 @@ public async Task AnalyzeSentimentWithOpinionMiningAsync()
                 "We changed rooms as the toilet smelled."
             };
 
-            AnalyzeSentimentResultCollection reviews = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { AdditionalSentimentAnalyses = AdditionalSentimentAnalyses.OpinionMining });
+            AnalyzeSentimentResultCollection analyzedReviews = await client.AnalyzeSentimentBatchAsync(reviews, options: new AnalyzeSentimentOptions() { MineOpinions = true });
```

Comment:
Let's call these analyzed reviews since they're the output of the analysis operation containing both the review information and the analysis info.